packages/web-fragments/src/gateway/fragment-gateway.ts

```diff
@@ -46,9 +46,28 @@ export class FragmentGateway {
 			);
 		}
 
+		// TODO: defensively deep clone the config object, but preserve any functions or property getters
+
 		// default to true
 		fragmentConfig.piercing ??= true;
 
+		if (fragmentConfig.iframeHeaders) {
+			// normalize header name casing to Camel-Case
```

Comment:
I suggest we say HTTP-Header-Case by convention, be cause camelCase is camelCase and it can be misunderstood?

Reply:
Sure

Reply:
done